crowd_anki/export/anki_exporter.py

```diff
@@ -48,21 +48,36 @@ def export_to_directory(self, deck: AnkiDeck, output_dir=Path("."), copy_media=T
                                        indent=4,
                                        ensure_ascii=False))
 
-        self._save_changes()
+        self._save_changes(deck)
 
         if copy_media:
             self._copy_media(deck, deck_directory)
 
         return deck_directory
 
-    def _save_changes(self):
-        """Save updates that were maid during the export. E.g. UUID fields"""
-        # This saves decks and deck configurations
-        self.collection.decks.save()
-        self.collection.decks.flush()
+    def _save_changes(self, deck, is_export_child=False):
```

Comment:
does this preserve changes to note uuid btw?

Reply:
Do you mean note model uuid? (If so, yes, I think so.) (Sorry, I might not have fully woken up yet...)

Reply:
no, I meant actual note uid. but it seems it falls back on using the native guid, nvm :p